app/src/main/java/com/work/restaurant/view/calendar/CalendarFragment.kt

```diff
@@ -0,0 +1,121 @@
+package com.work.restaurant.view.calendar
+
+import android.app.DatePickerDialog
+import android.os.Build
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.DatePicker
+import android.widget.TextView
+import android.widget.Toast
+import com.work.restaurant.R
+import com.work.restaurant.util.App
+import com.work.restaurant.view.base.BaseFragment
+import kotlinx.android.synthetic.main.calendar_main.*
+import java.text.SimpleDateFormat
+import java.util.*
+
+class CalendarFragment : BaseFragment(R.layout.calendar_main),
+    View.OnClickListener {
+
+
+    override fun onClick(v: View?) {
+
+        when (v?.id) {
+
+        }
+    }
+
+    var button_date: Button? = null
+    var textview_date: TextView? = null
+    var cal = Calendar.getInstance()
```

Comment:
```suggestion
    private val cal = Calendar.getInstance()
```
외부에서 참조할 경우와 다시 init될일이 없을 것 같아요

Reply:
e9c0f82 수정하였습니다.